satpy/readers/__init__.py

```diff
@@ -538,3 +539,61 @@ def _get_reader_kwargs(reader, reader_kwargs):
         reader_kwargs_without_filter[k].pop('filter_parameters', None)
 
     return (reader_kwargs, reader_kwargs_without_filter)
+
+
+@total_ordering
+class FSFile(os.PathLike):
+    """Implementation of a PathLike file object, that can be opened.
+
+    This is made to be use in conjuction with fsspec or s3fs. For example::
+
+        zipfile = S3B_OL_2_WFR____20201103T100807_20201103T101107_20201103T121330_0179_045_179_1980_MAR_O_NR_002.zip
```

Comment:
There appear to be some string-delimiting quotes missing here.

Reply:
indeed thanks.